libs/langchain/langchain/output_parsers/openai_tools.py

```diff
@@ -22,6 +22,8 @@ class JsonOutputToolsParser(BaseGenerationOutputParser[Any]):
     """
     return_id: bool = False
     """Whether to return the tool call id."""
+    return_single: bool = False
```

Comment:
This doesn't feel like an option that we should add to the parsers

Reply:
why not

Reply:
Do you mind if we re-name this to: "first_tool_only" so it's clear how to interpret "single"